src/spawn/index.ts

```diff
@@ -0,0 +1,300 @@
+import {Identity, Application} from 'openfin/_v2/main';
+import {ChannelClient} from 'openfin/_v2/api/interappbus/channel/client';
+import {_Window} from 'openfin/_v2/api/window/window';
+import {ApplicationOption} from 'openfin/_v2/api/application/applicationOption';
+import {WindowOption} from 'openfin/_v2/api/window/windowOption';
+import {ConfigWithRules} from 'openfin-service-config';
+
+export type Dictionary<T = string> = {
```

Comment:
I know this is probably not yours, but Typescript has a built-in `Record` type defined as 
```ts
/**
 * Construct a type with a set of properties K of type T
 */
type Record<K extends keyof any, T> = {
    [P in K]: T;
};
```
which should cover this use case with `Dictionary<T> = Record<string, T>`.

Might be worth swapping it out while we're in this file.

Reply:
Yup not mine but I can make that switcheroo for you

Reply:
🥇

Reply:
Done!